Makefile

```diff
@@ -16,11 +17,15 @@ fmt:
 test:
 	go test -v ./...
 
-test-docker: check-ceph-version .build-docker
+.PHONY: test-docker test-container
+test-docker: test-container
```

Comment:
I'm fine with dropping any references to docker in targets. Just `test-container` and a `.container-id` file would be cleaner.

But, that can be done later as well.

Reply:
Yeah, I didn't want to break anyone's current workflow.

I plan on changing what's currently ".build-docker" more in the future because I find it annoying when I need to locally test different versions and that forces a rebuild. But that's for a future PR.